README.md

```diff
@@ -83,6 +83,5 @@ Subscribing can be done through the [mailman web interface][mailman] or by
 [csi_addons_spec]: https://github.com/csi-addons/spec/
 [csi_addons]: https://csi-addons.github.io/
 [csi]: https://kubernetes-csi.github.io/docs/
-[operatorhub]: https://operatorhub.io/
```

Comment:
why this is removed?

Reply:
We had this error here, MD053/link-image-reference-definitions Link and image reference definitions should be needed [Unused link or image reference definition: "operatorhub"] [Context: "[operatorhub]: https://operato./.."]

Since there was no reference for operator hub in the doc, so I removed it. Is this fine or I shall add a comment to ignore this error here?